javaparser-core/src/main/java/com/github/javaparser/printer/DefaultPrintableSource.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (C) 2011, 2013-2020 The JavaParser Team.
+ *
+ * This file is part of JavaParser.
+ *
+ * JavaParser can be used either under the terms of
+ * a) the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ * b) the terms of the Apache License
+ *
+ * You should have received a copy of both licenses in LICENCE.LGPL and
+ * LICENCE.APACHE. Please refer to those files for details.
+ *
+ * JavaParser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+
+package com.github.javaparser.printer;
+
+import static com.github.javaparser.printer.configuration.Indentation.IndentType.SPACES;
+
+import java.util.Deque;
+import java.util.LinkedList;
+
+import com.github.javaparser.Position;
+import com.github.javaparser.printer.configuration.ConfigurablePrinter;
+import com.github.javaparser.printer.configuration.Indentation;
+import com.github.javaparser.printer.configuration.Indentation.IndentType;
+import com.github.javaparser.printer.configuration.PrinterConfiguration.ConfigOption;
+import com.github.javaparser.utils.Utils;
+
+/**
+ * A support class for code that outputs formatted source code.
+ */
+public class DefaultPrintableSource implements PrintableSource {
```

Comment:
Naming: this is the default for source that is printable? I don't know what that means.

Reply:
DefaultPrintableSource class is  the same as actual SourcePrinter class (except for the initialization with configuration)

Reply:
Not sure this is a need for the interface PrintableSource because it's a implementation detail. I will remove it.

Reply:
Interface printableSource and class DefautlPrintableSource are removed because there are useless.